types/highcharts/index.d.ts

```diff
@@ -4644,6 +4644,14 @@ declare namespace Highcharts {
          * allows setting a separate color for the line without altering the fillColor.
          */
         lineColor?: string | Gradient;
+        /**
+         * For area-like series, allow the font size to vary so that small areas get a smaller font size. The default applies this effect to area-like series but not line-like series.
+         */
+        maxFontSize?: number;
```

Comment:
In the [docs link you posted](https://api.highcharts.com/highcharts/series.line.label.maxFontSize), it states that the default value is `null`. Should the type therefore be `number | null` in each of these cases?